src/utils/DragAndDrop.js

```diff
@@ -183,6 +183,13 @@ define(function (require, exports, module) {
             event.preventDefault();
 
             options.ondragleave(event);
+            // selecting FILE bar 
+            // adding functionality when dragging files away from FILE bar
+            // make call to hoverOver function to change styles
+            var sidebar = document.getElementById("sidebar");
```

Comment:
In this code you're mixing pure DOM js with jquery, and we might as well settle on one and stick with it.  Brackets uses jquery everywhere, so you can go ahead and use it.  Here, I'd modify your code like so:

```js
$("#sidebar").on("dragleave", function() {
    $("#sidebar").removeClass("sidebar-hover");
});
```